projects/plugins/jetpack/extensions/shared/memberships/settings.js

```diff
@@ -133,7 +133,7 @@ export function NewsletterAccessRadioButtons( {
 	postHasPaywallBlock: postHasPaywallBlock = false,
 } ) {
 	const isStripeConnected = stripeConnectUrl === null;
-	const { emailSubscribers, paidSubscribers } = useSelect( select =>
+	const { totalSubscribers, paidSubscribers } = useSelect( select =>
```

Comment:
For

![image](https://github.com/user-attachments/assets/cd65cccf-f2ec-493b-b949-c79355b5a097)